blog/_posts/2016-09-30-The-end-of-an-amazing-summer(winter) of code.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The end of an amazing summer(winter) of code.
+layout: post
+created_at: Fri sept 30 2016
+permalink: blog/2016-09-30-The-end-of-an-amazing-summer(winter) of code.
+current: blog
+author: Ramya
+twitter: ramya690
+categories:
+- student-posts
+---
+
+The Rails Girls Summer of Code is one of  the best things that happened to me. After finishing my course at the university I got an opportunity to take part in the program representing the Melbourne team.
+I was very happy and excited by the fact that I will be contributing to an Open source project for 3 months under the guidance of all the coaches and [Katrina Owen](https://twitter.com/kytrinyx) as a mentor. And yes, it was kicked off on 1st of July!
+
+I worked on the project [exercism](http://exercism.io/) and contributed to both x-api and its web client and following summarizes the skills that I have gained over the past three months.
+
+**Ruby**: Prior to this, I had worked on other object oriented programming language like Java. The biggest challenge for me was picking up Ruby and writing clean and elegant ruby code. Thanks to all coaches I got improved by the end of the program.
+
+**GIT**: I learnt many useful techniques about GIT like squashing the commits, cherry pick the commits, rebasing and more. It was useful to know how important to write meaningful commit messages, titles and descriptions for PR especially when working on open source projects.
+
+**Test driven development**: I learnt Minitest testing framework when working on different issues. Writing tests first helped me understand what I really want from the code. It ensures changes and new code don’t break the expectations of existing code. I had a good fun writing code to pass tests and then refactor the code and see if it still passes my test.
+
+
+Working with people geek company like Culture Amp I learnt many things. Attending their culture x design was very useful. Few take away points:
+
+ - Stay positive at the work, including with the people that are working with us.
+ - Treating coworkers with respect and do our best to see    issues from
+   their point of view.
+ - Being open to feedback  can give us an awareness of blind spots, help us to learn, and become better at what we do.
+
+It was a great pleasure working with these amazing minds.
+
+<img src="/img/blog/2016/goodbye-from-vegemite.jpg" alt="Team Vegemite">
+<br><font color="grey"><small><i>With Coaches (L-R): Kevin, Ramya, Jo & Tim</i></small></font>
+
+Overall it was a great program once again! Setting the personal goals and working towards them was the only key I followed. At the end, I am employed as a full-time software engineer. I would like to thank Rails Girls Summer of Code for giving me this opportunity and each and everyone who have helped and supported me during this journey.
+
+<img src="/img/blog/2016/vegemite-farewell.jpg" alt="Team Vegemite">
```

Comment:
the image credits (i.e. who the photo was taken by or "belongs to") are still missing.